.claude/skills/integration-pipeline-preview/references/nhcore-context.md

```
@@ -0,0 +1,65 @@
+
+- [Toolchain invocation](#toolchain-invocation)
+- [Global compiler flags](#global-compiler-flags)
+- [Custom Prelude](#custom-prelude)
+- [Framework-provided defaults](#framework-provided-defaults)
+- [Red-flag-but-already-handled list](#red-flag-but-already-handled-list)
+
+## Toolchain invocation
+
+Every Haskell-toolchain command (`cabal`, `hlint`, `ghc`, `hspec-discover`, etc.) must be invoked through the repo's nix dev shell. The canonical form is:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)